src/main/java/com/google/sps/servlets/utils/KeyRetriever.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.utils;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class KeyRetriever {
+  private static String keyPath = System.getProperty("user.dir") + "/apiKey";
+
+  public static String getApiKey() {
+    try {
+      return Files.readAllLines(Paths.get(keyPath), StandardCharsets.US_ASCII).get(0);
```

Comment:
Just curious: what is the get(0) for?

Reply:
So I think for whatever reason readAllLines returns a list, since the Paths.get() returns all file paths that match the query being entered. So, if there were mutliple files pertaining to the pattern of the keypath, it would return all of them.

Reply:
readAllLines returns a list of lines.

Reply:
Oh you're right, I don't know what I had confused myself like that for.
The reason it returns readAllLines.get(0) is effectively to convert the list of Strings of length 1 into a single String